internal/release/render/render.go

```diff
@@ -0,0 +1,178 @@
+package render
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"html/template"
+	"net/url"
+	"os"
+	"path/filepath"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/nestoca/joy/api/v1alpha1"
+	"github.com/nestoca/joy/internal"
+	"github.com/nestoca/joy/internal/environment"
+	"github.com/nestoca/joy/internal/helm"
+	"github.com/nestoca/joy/internal/release/cross"
+	"github.com/nestoca/joy/pkg/catalog"
+	"gopkg.in/yaml.v3"
+)
+
+type RenderParams struct {
+	Env          string
+	Release      string
+	DefaultChart string
+	CacheDir     string
+	Catalog      *catalog.Catalog
+	IO           internal.IO
+	Helm         helm.PullRenderer
+}
+
+func Render(ctx context.Context, params RenderParams) error {
+	environment, err := getEnvironment(params.Catalog.Environments, params.Env)
+	if err != nil {
+		return fmt.Errorf("failed to get environment: %w", err)
```

Comment:
When wrapping error, let's use positive format:
```suggestion
		return fmt.Errorf("getting environment: %w", err)
```
Same thing for all other errors.

Reply:
I find this positive format wrapping to be unclear. Its more terse and reads less nice, and you need to have some top level initial first message be the "negative" message but its hard to say where that would happen.

If you look at the errors in the tests they are clear such as:
```
failed to get environment: not found: quality-assurance
failed to get release: not found within environment qa: app
```

I think the positive wrap will make them unclear. Like a sentance without a beginning.

Reply:
We already discussed that point in length before your coming and settled on that style, so please let's keep on sticking to it. FYI, the idea is that when you have deeply nested error messages, they become very redundant and verbose, with no added value.  Usually, errors are already displayed with "error: ...", and we would get something like "error: failed to load template file: failed to parse line 5: failed to parse unexpected character ':' in token 'asdf:asdf'", but rather have the more straight to the point version: "error: loading template file: parsing line 5: unexpected character ':' in token 'asdf:asdf'"

Reply:
Will change it